src/vs/editor/standalone/browser/standaloneServices.ts

```diff
@@ -1100,6 +1101,7 @@ registerSingleton(IClipboardService, BrowserClipboardService, InstantiationType.
 registerSingleton(IContextMenuService, StandaloneContextMenuService, InstantiationType.Eager);
 registerSingleton(IMenuService, MenuService, InstantiationType.Eager);
 registerSingleton(IAudioCueService, StandaloneAudioService, InstantiationType.Eager);
+registerSingleton(IInputResultsService, InputResultsService, InstantiationType.Eager);
```

Comment:
Does it have to be Eager? I think it will get instantiated when one of the view parts pulls it in anyway

Reply:
Fix in commit https://github.com/microsoft/vscode/pull/187077/commits/af2e8b585a16e3e75af38bef10daf6231792fce2